docs/1_how_it_works/1_1_how_it_works.md

```diff
@@ -0,0 +1,51 @@
+# How Finetuner Works
+
+## Contrastive Metric Learning 👓
+
+From the algorithmic perspective,
+**Finetuner** leverage contrastive metric learning approach to improve your model.
+How it works?
+
+### Step 1: Convert your model into an Embedding Model
+
+Finetuner interpret your model architecture,
+remove the default *head*, apply *pooling* and freeze layers not necessary to train.
+For example, if your pre-trained model is trained for cat-dog classification.
+Finetuner is going to remove the cat-dog classifier and turn your model into an *embedding model*.
+
+This *embedding model* do not make predictions and output a probability,
```

Comment:
"This *embedding model* does not make predictions or outputs a probability, but instead outputs a *feature vector* to represent your data.